src/imageLoader/wadors/metaData/fixNMMetadata.js

```diff
@@ -0,0 +1,16 @@
+export default function fixNMMetadata(metaData) {
+  if (!metaData['00200032'] || metaData['00200037']) {
+    // adjust metadata in case of multiframe NM data, as the diucom tags
+    // 00200032 and 00200037 could be found only in the dicom tag 00540022
+    if (
```

Comment:
Please use a nested conditional operator here, eg:
if ( metaData['00540022']?.Value?.length > 0

Reply:
Better yet, use the getTagValue from above.

Reply:
Done